web-console/Storage/OCS/OCS _Installation_Workflow.md

```diff
@@ -0,0 +1,119 @@
+# OCS Install Workflow
+
+### OperatorHub
+
+There will be 2 OCS operator entries in OperatorHub covering each of the following modes. 
+* Converged Mode -- Local Cluster / Hyperconverged 
+* Independent Mode -- External Cluster (Ceph Cluster is deployed externally)
+
+### Converged Mode Design
+
+OperatorHub
+![OperatorHub operators view](img/OCS-operator-view.png)
+
+Installation
+![OperatorHub operators view](img/OCS-Install-step1.png)
+
+![OperatorHub operators view](img/OCS-Install-step2.png)
+
+After the subscription process, the user will see the subscribed operator page. 
+
+![subscription operators page](img/OCS_Subscription_page.png) 
+
+In the installed operator view the user can see the OCS operator installed. 
+
+![OperatorHub operators view](img/Installed_operators_list.png)
+
+
+# The OCS installation process
+OCS Overview page
```

Comment:
@shirimordechay Between the OCS_Subscription_page.png and OCS-View.png, it's missing a couple other mockups of what that workflow now looks like in OperatorHub.

Also for OCS-View.png, the "OCS" and "MCG" tabs should also exist before user creates the OCS service based on the latest OCP 4.1 OperatorHub deployment workflow.

Please update accordingly.  Thanks.

@yuvalgalanti FYI.

Reply:
Which mockups are missing in this flow?

Reply:
@julienlim Open question for you here ^

Reply:
@shirimordechay @lizsurette @tlwu2013 

Overall it LGTM.  Just the operator install/subscription didn't quite seem to match what @tlwu2013 had shared recently (https://projects.invisionapp.com/share/7TRU0EYHJNM#/screens/361317773) with me but does match closer to what's in OCP currently.

As long as @tlwu2013 is ok with it, we should be fine. :-)